Y/yaml_cpp/build_tarballs.jl

```diff
@@ -32,11 +40,12 @@ platforms = expand_cxxstring_abis(supported_platforms(; experimental=true))
 
```

Comment:
Should cxx string ABIs only be expanded on Linux?

Reply:
No, also on Windows

Reply:
OK